.happy/terraform/modules/lambda/variables.tf

```diff
@@ -18,6 +18,11 @@ variable cellxgene_bucket {
   description = "Cellxgene bucket name"
 }
 
+variable dataset_submission_bucket {
```

Comment:
I think the dataset submission bucket should be an output (as is currently the case for frontend url, backend url...). I presume this would result in the s3 bucket automatically being logged out at the end of an rdev apply.